saltproc/tests/saltproc_test.py

```diff
@@ -79,26 +103,45 @@ def test_write_mat_file():
                 assert line.rstrip() == solution
 
 
-def test_process_fuel():
```

Comment:
why is this test deleted? Is it because this PR has removed this functionality? Has equivalent functionality replaced this? Will previous simulations run properly? 

Tests shouldn't just be deleted because they don't fit the current mood. A codebase need to move forward, not back and forth.